pydra/tasks/mriqc/workflows/anatomical/base.py

```diff
@@ -982,33 +956,32 @@ def airmsk_wf(name="AirMaskWorkflow"):
         ),
         name="inputnode",
     )
-    outputnode = pe.Node(
-        niu.IdentityInterface(fields=["hat_mask", "air_mask", "art_mask", "rot_mask"]),
+    outputnode = pydra.Node(
+        interface=specs.IdentityInterface(fields=["hat_mask", "air_mask", "art_mask", "rot_mask"]),
         name="outputnode",
     )
 
-    rotmsk = pe.Node(RotationMask(), name="RotationMask")
-    qi1 = pe.Node(ArtifactMask(), name="ArtifactMask")
+    rotmsk = ShellCommandTask(name="RotationMask", executable="RotationMask")
+    qi1 = ShellCommandTask(name="ArtifactMask", executable="ArtifactMask")
 
-    # fmt: off
-    workflow.connect([
-        (inputnode, rotmsk, [("in_file", "in_file")]),
-        (inputnode, qi1, [("in_file", "in_file"),
-                          ("head_mask", "head_mask"),
-                          ("ind2std_xfm", "ind2std_xfm")]),
-        (qi1, outputnode, [("out_hat_msk", "hat_mask"),
-                           ("out_air_msk", "air_mask"),
-                           ("out_art_msk", "art_mask")]),
-        (rotmsk, outputnode, [("out_file", "rot_mask")])
-    ])
-    # fmt: on
+    workflow.add(rotmsk)
```

Comment:
This should work, but typically we add the node to the workflow as it is created